Resources/doc/reference/installation.rst

```diff
@@ -23,10 +23,9 @@ Installation
 
 Retrieve the bundle with composer:
 
-.. code-block:: sh
-
-    php composer.phar require sonata-project/media-bundle --no-update
+.. code-block:: bash
 
+    $ php composer.phar require sonata-project/media-bundle --no-update
```

Comment:
you are right, we should change this, but we should do this across all the Sonata bundles